packages/web/src/scss/components/Pagination/_theme.scss

```diff
@@ -0,0 +1,21 @@
+@use 'sass:map';
+@use '@tokens' as tokens;
+
+$item-gap: tokens.$space-300;
+$item-min-width: 2em;
+$item-height: 2.5em;
+$item-padding-inline: tokens.$space-400;
```

Comment:
I think padding inline should be 12px (500 token)? 
![image](https://user-images.githubusercontent.com/7041807/191764992-9760b699-c549-4fb0-82ee-abdb462e709b.png)

Reply:
There are different paddings for single digit value and multi digit value (meaning two digits). Padding is being used for two digits size (smaller) in combination with min-width that should make the trick that there is not so big difference between sizes of all elements in the row.

Reply:
Ok, maybe check this with designers or just do the UX check before merging.

Reply:
@crishpeen if you would check another single digit item you would see different padding.

I have consulted this with designers and this solution is OK with them. 

Items have min-width set to 2em and padding to 8px. That means that single digits are always 32px wide and two/three digit items have floating width according to font rendered value respectively.